src/platform/utilities/ui/Breadcrumbs.jsx

```diff
@@ -0,0 +1,118 @@
+// TODO: Move to Jean Pants when approved
+import PropTypes from 'prop-types';
+import React from 'react';
+import _debounce from './debounce';
+import uniqid from 'uniqid';
+
+class Breadcrumbs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      mobileShow: false,
+    };
+
+    this.id = this.props.id || uniqid('va-breadcrumbs-');
+    this.listId = this.props.listId || uniqid('va-breadcrumbs-list-');
+    this.mobileWidth = this.props.mobileWidth || 425;
+
+    this.debouncedToggleDisplay = this.debouncedToggleDisplay.bind(this);
+    this.renderBreadcrumbLinks = this.renderBreadcrumbLinks.bind(this);
```

Comment:
We do this both ways in our codebase, but I think the current preference is to use the auto bind syntax:

```js
someMethod = () => {
  ...
}
```